moma_demo:
  base_frame_id: panda_link0
  ee_grasp_offset_z: -0.04
  scan_joints:
    - [0.010, -1.479, 0.100, -2.416, 0.089, 1.402, 0.851]
    - [-0.520, -0.796, 0.912, -1.672, 0.091, 1.281, 0.871]
    - [0.032, 0.459, 0.074, -0.839, 0.015, 0.777, 0.833]
    - [0.092, -0.626, -0.551, -1.837, 0.165, 1.238, 0.900]
  drop_joints: [-0.329, -0.285, -0.289, -1.878, -0.070, 1.635, 0.801]

  task_frame_id: task
  table_height: 0.15  # Measured from the base frame specified above

  grasp_selection: "auto"  # Choice: manual, auto

  arm_velocity_scaling_drop: 0.3
  arm_velocity_scaling_grasp: 0.2

  object_type: "cubes"

  battery_lv: 1000
  battery_drop_rate: 5
  battery_charge_rate: 20
  battery_station: [0.5, -2.0, 0, 0, 0, 0, 1]

  # Waypoints for the base to follow during search action:
  search_waypoints:
    - [-1.0, -1.0, 0, 0, 0, 1, 0]
    - [-1.0, 1.9, 0, 0, 0, 1, 0]
    - [0.5, 2.1, 0, 0, 0, 0.7071068, 0.7071068]
    - [2.0, 1.5, 0, 0, 0, 0, 1]
    - [2.0, -1.0, 0, 0, 0, 0, 1]
  
  delivery_station: [0.5, 2.1, 0, 0, 0, 0.7071068, 0.7071068]
  place_pose: [0.5, 0.0, 0.25]
  inspection_station: [-1.0, -1.0, 0, 0, 0, 1, 0]

vgn:
  model: $(find vgn)/assets/models/vgn_conv.pth

tsdf_server:
  frame_id: task
  length: 0.3
  resolution: 40
  depth_scaling: 1.0  # Set to 1.0 for Gazebo, 0.001 for a RealSense
  camera:
    frame_id: wrist_camera_depth_optical_frame
    info_topic: /wrist_camera/color/camera_info
    depth_topic: /wrist_camera/depth/image_rect_raw
